bigtable-hbase-1.x-replication/src/main/java/com/google/cloud/bigtable/hbase/replication/metrics/HBaseMetricsExporter.java

```diff
@@ -0,0 +1,20 @@
+package com.google.cloud.bigtable.hbase.replication.metrics;
+
+import org.apache.hadoop.hbase.replication.regionserver.MetricsSource;
+
+/**
+ * HBaseMetricsExporter implements MetricExporter which bridges with MetricsSource.
+ */
+public class HBaseMetricsExporter implements MetricsExporter {
+    // set this as static
+    private static MetricsSource metricsSource;
```

Comment:
Why does this need to be static? I am not sure this is a good idea?  If hbase context's metric source is not static, please keep this as non-static. 

If you decide to keep it static, make sure its threadsafe. There can be more than one threads accessing this field.